src/modules/storage/Storages.ts

```diff
@@ -20,27 +20,32 @@ export class Storages
     /**
      *  The instance of sqlite3
      */
-    protected db: sqlite.Database;
+    protected db: any;
```

Comment:
Why any?  `sqlite.Database`

Reply:
I made the type 'any' because if the member variable is not generated by the constructor, the error occurs.

Reply:
That's not an improvement.

Reply:
I change to `protected db: sqlite.Database|any;`